gammapy/cube/tests/test_fit.py

```diff
@@ -97,4 +108,4 @@ def test_cube_fit(sky_model, counts, exposure):
                              rtol=1e-2)
 
     stat = np.sum(fit.stat, dtype='float64')
-    assert_allclose(stat, 13079.031915029665)
+    assert_allclose(stat, 8.672365798603572)
```

Comment:
Model best fit parameter didn't change (which is expected I guess). Don't know where this change in fit stats comes from ...

Reply:
Probably a few more asserts would help to see where changes come from moving forward.
Especially on npred for one or a few well-chosen pixels.